README.md

```diff
@@ -1,15 +1,37 @@
 <h1 align="center"> TkTerminal </h1>
-A terminal emulator written in Python using tkinter
 
-#### 🌏 [简体中文](README_CH.md)
+[![PyPI](https://img.shields.io/pypi/v/tktermwidget)](https://pypi.org/project/tktermwidget)
+![Platform](https://img.shields.io/powershellgallery/p/Pester?color=blue)
 
-## Views:
+### 🌏 [简体中文](README_CH.md)
+
+```TkTerminal``` is a terminal emulator written in Python using tkinter. It is quite easy to use.
 ### Windows
 <img src="images/windows.png" width="75%" align="center">
 
 ### MacOS
 <img src="images/macos.png" width="85%" align="center">
 
+## Features
+- User can set the terminal widget with their own options
+- Use \ to make new lines (On Windows it is &&)
+- Command history recorder
+- Styles
+- And some on
+
+## Future ideas
+- Highlight
+
+## Styles
```

Comment:
How can I make my own style or have it conform to the system settings?

Reply:
```
Command = {  # Style for normal "cmd.exe"
    "background": "#000000",
    "insertbackground": "#f2f2f2",
    "selectbackground": "#f3f3f3",
    "selectforeground": "#000000",
    "foreground": "#f2f2f2",
}

# TODO: add a user custom style function later...
```
This is a todo. Maybe improve it in anthor pr.
My idea to do it:
- Write a gui to set these things
- Save it to a json file
- Read the style in the json file
- Add the style

Reply:
If it's just a dictionary couldn't you just explain how to write them ? Couldn't you make the styles override and all kwargs too instead of just these?

Reply:
```

# Styles format:
# {yourstylename}: dict = {
#    "background": "{yourhexcolor}",
#    "insertbackground": "{yourhexcolor}",
#    "selectbackground": "{yourhexcolor}",
#    "selectforeground": "{yourhexcolor}",
#    "foreground": "{yourhexcolor}",
# }
```
Okay I wrote them in the #45 pr